README.md

```diff
@@ -1,376 +1,242 @@
+### Browser support
 
-You can employ `postcss-transformselectors` to change the classnames Spectrum CSS uses. For instance, you may want to use bare `h1`/`h2`/`h3` instead of `.spectrum-Heading.spectrum-Heading--size*`.
+We maintain a relatively modern codebase that supports the latest two versions of evergreen web browsers. The current list of browsers officially supported by Spectrum CSS can be found in the browserslist section of the project's package.json file. This setting is used by the build tools when the source files are built. If you require additional browser support for your project, the CSS can processed further with your chosen tools.
 
-To do this, first install the plugin:
+* latest 2 Edge versions
+* latest 2 Chrome versions
+* latest 2 Firefox versions
+* latest 2 Safari versions
+* latest 2 iOS versions
 
-```js
-npm i postcss-transformselectors --save-dev
-```
+### Optimizing Spectrum CSS
 
-Then, add something like this to your `postcss.config.js`:
-
-```js
-module.exports = {
- plugins: [
-  require("postcss-transformselectors")({
-   replace: [
-    { search: ".spectrum-Heading--sizeXXL", replace: "h1" },
-    { search: ".spectrum-Heading--sizeXL", replace: "h2" },
-    { search: ".spectrum-Heading--sizeL", replace: "h3" },
-   ],
-   transform: (selector) => {
-    if (selector.startsWith(".spectrum-Heading")) {
-     // Operate on each selector in a selector list
-     return selector
-      .split(",")
-      .map((selectorPart) => {
-       // Create separate selectors for each reference to .spectrum-Heading
-       return ["h1", "h2", "h3"]
-        .map((h) => {
-         return selectorPart.replace(".spectrum-Heading", h);
-        })
-        .join(",");
-      })
-      .join(",");
-    }
-
-    // Don't mess with things that don't have .spectrum-Heading in them
-    return selector;
-   },
-  }),
- ],
-};
-```
+Spectrum CSS is designed to be as modern and flexible as possible, and as such, leaves room for consumers to optimize in their own way. There are many tools and techniques you can use to optimize CSS for your project such as tree shaking, purging, and minification. If you are loading the entire set of `@spectrum-css/tokens` for example, you can safely tree shake the tokens to only include the variables you are using, often leading to a significant reduction in file size.
 
 ## Contributing
 
 A very special thank you to all of our [contributors](<https://github.com/adobe/spectrum-css/graphs/contributors>) without whom this project would not be possible.
 
 Want to join the team? Check out the [contributing guidelines](.github/CONTRIBUTING.md) for quick start information.
 
-### Building
+### Getting started
 
-Run the following commands:
+To get started with Spectrum CSS, you'll need to clone the repository and install the dependencies.
 
 ```shell
 yarn install
 ```
 
-**Important:** Requires at least Node 18.x.
+**Important:** Requires >= Node 18.8.1 and Yarn 1.22.0.
```

Comment:
Maybe we could add something about `nvm` and running `nvm use` during local development to use the required version of Node defined in the `.nvmrc` file? This would go above the instruction to `yarn install`.